pkg/cache/placeholder.go

```diff
@@ -46,6 +46,11 @@ type Placeholder struct {
 }
 
 func newPlaceholder(placeholderName string, app *Application, taskGroup v1alpha1.TaskGroup) *Placeholder {
+	ownerRefs := app.placeholderOwnerReferences
+	controller := false
+	for _, r := range ownerRefs {
+		*r.Controller = controller
+	}
```

Comment:
we are setting the controller field to false all the time.
could you pls add some more comments why we are doing this?